### DISCUSSION TITLE
Why do my logs show values and warnings for cell voltages and resistances that I have removed from the YAML code?

### DISCUSSION THREAD

**Discussion by austin202220** [syssi/esphome-jk-bms#250] [Q&A] [answered]
My log level is INFO and I have removed all of the wire resistance entities and also some of the cell voltages (I only use 16 out of the available 24 cells for this BMS). Why are the logs showing me values and warnings for cells that dont exist?

yaml code attached via a text file, couldn't get it to display nicely on the post.

<img width="1498" alt="Screenshot 2023-02-16 at 10 41 37 AM" src="https://user-images.githubusercontent.com/97492072/219431732-1c1ee9aa-ac03-4b4d-9133-6c56fdde2ac6.png">
<img width="1502" alt="Screenshot 2023-02-16 at 10 41 54 AM" src="https://user-images.githubusercontent.com/97492072/219431749-324b1992-9e99-441f-8e3e-267e77804b93.png">

[logs_jk-bms-esp32-01_run.txt](https://github.com/syssi/esphome-jk-bms/files/10758352/logs_jk-bms-esp32-01_run.txt)

[jk-bms-esp32-01 yaml.txt](https://github.com/syssi/esphome-jk-bms/files/10758357/jk-bms-esp32-01.yaml.txt)

**Comment by syssi** [comment, marked as answer]
This is the decoded settings frame / BMS configuration retrieved once per `retrieve_settings` call. You could removed the periodic retrieval of the settings.

**Reply by austin202220** [reply]
ok thanks, I get a constant "wire resistance warning: 00 00 00 00" message, do you know why? My BMS setting is set to 16 cells and all 16 have good resistance values, I shouldn't be getting any warnings

**Reply by syssi** [reply]
You can ignore this output. Zeros are good.

**Reply by syssi** [reply]
I've improved the code / labels to avoid this confusion in future: https://github.com/syssi/esphome-jk-bms/pull/251/files
